docs/002-projects.apib

```diff
@@ -337,3 +337,94 @@ Accept or decline a transfer request.
 ### Decline a Transfer Request [DELETE]
 
 + Response 204
+
+
+## IP Reservations [/projects/{id}/ips]
+Provides a list of IP resevations for a single project.
+
++ Parameter
+  + id (string) ... The project ID.
+
+### Retrieve all IP reservations [GET]
+
++ Response 200
+
+  + Body
+
+            [
+              {
+                "id": "174de894-01d3-45ba-b241-f24679d6f00c",
+                "address": "147.75.193.9",
+                "network": "147.75.193.9",
+                "address_family": 4,
+                "public":true,
+                "cidr":32,
+                "assignments": [
+                                  { "href": "/ips/7ab2c92c-4db4-4bb5-aaac-471cb8cb55c4" },
+                                  { "href": "/ips/ee783f17-4bd3-4798-893f-5c3d95e67b1c" }
+                                ]
+                ...
+              },
+              {
+                "id": "174de894-01d3-45ba-b241-f24679d6f00d",
+                "address": null,
+                "network": "2604:1380:4:1300::",
+                "address_family": 6,
+                "public":false,
+                "cidr":31,
+                "assignments": [
+                                  { "href": "/ips/7ab2c92c-4db4-4bb5-aaac-471cb8cb55c4" },
+                                  { "href": "/ips/ee783f17-4bd3-4798-893f-5c3d95e67b1c" }
+                                ]
+                ...
+              }
+            ]
+
+### Requesting IP reservations [POST]
+Request more IP space for a project in order to have additional IP addresses to assign to devices
+
++ Request
+
+  + Body
+
+            {
+              "type": "public_ipv4", // public_ipv4 || global_ipv4
+              "quantity": 8, // How many IPv4 you want to request. Options: 1, 2, 4, 8, 16, 32, 64, 128, 256
+              "comments": "This is my comment to the Packet team" // Optional comment
+            }
+
++ Response 204
+
+## IP Reservation [/ips/{id}]
+In order to see information about an IP reservation.
```

Comment:
See the wording comments on `/devices/:id/ips` above--just remove the "In order to" part.